templates/overrides/search/form-element--oe-whitelabel-search-form.html.twig

```diff
@@ -0,0 +1,30 @@
+{#
+/**
+ * @file
+ * Theme implementation for the search form element.
+ *
+ * Display search form without wrappers or other elements like label, prefix, suffix or description.
+ *
+ * @see ./core/modules/system/templates/form-element.html.twig
+ */
+#}
+{% set classes = [
+  'js-form-item',
+  'form-item',
+  'js-form-type-' ~ type|clean_class,
+  'form-item-' ~ name|clean_class,
+  'js-form-item-' ~ name|clean_class,
+  title_display not in ['after', 'before'] ? 'form-no-label',
+  disabled == 'disabled' ? 'form-disabled',
+  errors ? 'form-item--error',
+] %}
+
+{{ children }}
+
+{% if errors %}
+  <div{{ attributes.addClass(classes) }}>
+    <div class="form-item--error-message invalid-feedback d-block">
+      {{ errors }}
+    </div>
+  </div>
+{% endif %}
```

Comment:
How can we test this part with `errors`?
I tried different tricks to submit invalid values, but the errors section never shows up.

The section seems weird to me, because all the classes would only make sense for a complete form element, not just for a section with errors.

Reply:
If we don't care about the "errors" section, and only want to get rid of wrappers, then we could simply set `''#theme_wrappers' => []` in the render element.
We could do this in the form builder class itself, or in the block plugin class. The question is only if we want to allow sub-themes to override it, or make the block plugin and the form builder class agnostic of the whitelabel theme.
TBD.

Reply:
Found it:
https://www.drupal.org/project/drupal/issues/2578561
https://git.drupalcode.org/project/drupal/-/commit/e55a7d8dc32aae6029863ea8a55e48515354053e#fee981558c2f41b50b192c470cfda2c49bb3c29a

So there is an optional module `inline_form_errors` in core, which enables this functionality.

Reply:
Personally I would rather get rid of this `error` part, which would make the template fully empty except for `{{ children }}´.
Then we would set `$form['#disable_inline_form_errors'] = TRUE;` in the form, and not have to worry about inline errors even if `inline_form_errors` module is enabled.

However, I talked with @brummbar, and for now the conclusion is to keep the errors section in the template, just without the outer wrappers.
So only this part `<div class="form-item--error-message invalid-feedback d-block">...`.
I think the current version is taken from oebt, whereas the core version has fewer classes. To me it makes more sense to use the oebt version, given that this is our base class.

Either way it will look bad - but this shall be fixed later.

Reply:
I tried locally with `<div class="form-item--error-message invalid-feedback d-block">` as in form-element in oebt, and with `<div class="form-item--error-message">` as in core. None of them look really good, but I'd say the core version looks a bit cleaner. In the end it does not matter much: When people want to use these inline form errors, they can open a new issue.